pyccel/stdlib/ndarrays/ndarrays.h

```diff
@@ -63,9 +67,9 @@ typedef struct  s_ndarray
     /* number of dimensions */
     int32_t                 nd;
     /* shape 'size of each dimension' */
-    int32_t                 *shape;
+    long int                *shape;
     /* strides 'number of bytes to skip to get the next element' */
-    int32_t                 *strides;
+    long int                *strides;

```

Comment:
See above. Why has the type changed. This will lead to truncation warnings on windows computers